packages/ml/src/RefMaxwell/ml_RefMaxwell.cpp

```diff
@@ -31,25 +31,59 @@ using Teuchos::ArrayRCP;
 
 
 // ============================================================================
-int* FindLocalDiricheltAndInflowRowsFromOnesAndZeros(const Epetra_CrsMatrix & Matrix, int &numBCRows){
-  int *dirichletRows = new int[Matrix.NumMyRows()];
-  numBCRows = 0;
-  for (int i=0; i<Matrix.NumMyRows(); i++) {
+void FindLocalDiricheltLikeRowsFromOnesAndZeros(const Epetra_CrsMatrix & Matrix, const double rowsum_threshold,
```

Comment:
Dirichlet is mispelled ... is this consistent in the calls?

Reply:
D'oh